physical-activity-prediction.ipynb

```diff
@@ -0,0 +1 @@
name":"stdout","output_type":"stream","text":["[ 4 10  8 ...  7  8  3]\n"]}],"source":["print(result)"]},{"cell_type":"code","execution_count":26,"id":"e36e98e0","metadata":{"execution":{"iopub.execute_input":"2023-08-05T01:44:37.612404Z","iopub.status.busy":"2023-08-05T01:44:37.61165Z","iopub.status.idle":"2023-08-05T01:44:37.616428Z","shell.execute_reply":"2023-08-05T01:44:37.615631Z"},"papermill":{"duration":1.837241,"end_time":"2023-08-05T01:44:37.618458","exception":false,"start_time":"2023-08-05T01:44:35.781217","status":"completed"},"tags":[]},"outputs":[],"source":["y_test_rs = np.argmax(y_test, axis=-1)"]},{"cell_type":"code","execution_count":27,"id":"49227522","metadata":{"execution":{"iopub.execute_input":"2023-08-05T01:44:41.477349Z","iopub.status.busy":"2023-08-05T01:44:41.476614Z","iopub.status.idle":"2023-08-05T01:44:41.481313Z","shell.execute_reply":"2023-08-05T01:44:41.48054Z"},"papermill":{"duration":1.939073,"end_time":"2023-08-05T01:44:41.484075","exception":false,"start_time":"2023-08-05T01:44:39.545002","status":"completed"},"tags":[]},"outputs":[{"name":"stdout","output_type":"stream","text":["9\n"]}],"source":["print(y_test_rs)"]},{"cell_type":"code","execution_count":28,"id":"72b1f910","metadata":{"execution":{"iopub.execute_input":"2023-08-05T01:44:45.446971Z","iopub.status.busy":"2023-08-05T01:44:45.446273Z","iopub.status.idle":"2023-08-05T01:44:45.569371Z","shell.execute_reply":"2023-08-05T01:44:45.568125Z"},"papermill":{"duration":2.161727,"end_time":"2023-08-05T01:44:45.571926","exception":false,"start_time":"2023-08-05T01:44:43.410199","status":"completed"},"tags":[]},"outputs":[],"source":["y_pred_enc = pd.get_dummies(result)\n","y_test_enc = pd.get_dummies(y_test)"]},{"cell_type":"code","execution_count":29,"id":"6b54f780","metadata":{"execution":{"iopub.execute_input":"2023-08-05T01:44:49.311088Z","iopub.status.busy":"2023-08-05T01:44:49.310688Z","iopub.status.idle":"2023-08-05T01:44:49.698079Z","shell.execute_reply":"2023-08-05T01:44:49.696602Z"},"papermill":{"duration":2.293735,"end_time":"2023-08-05T01:44:49.700483","exception":false,"start_time":"2023-08-05T01:44:47.406748","status":"completed"},"tags":[]},"outputs":[{"name":"stdout","output_type":"stream","text":["0.9624047680565351\n"]}],"source":["from sklearn.metrics import accuracy_score\n","print(accuracy_score(y_test_enc, y_pred_enc))"]}],"metadata":{"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.10.12"},"papermill":{"default_parameters":{},"duration":5890.466662,"end_time":"2023-08-05T01:44:54.111787","environment_variables":{},"exception":null,"input_path":"__notebook__.ipynb","output_path":"__notebook__.ipynb","parameters":{},"start_time":"2023-08-05T00:06:43.645125","version":"2.4.0"}},"nbformat":4,"nbformat_minor":5}
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    y = dataset.iloc[:, 0].values
</pre><p><code>activity_id = dataset.activityID</code></p><p>Convert it to a list if you need to but generally pandas DSs are more efficient.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ManviNagdev/physical_activity_prediction/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>